models/ws/frame.lua

```diff
@@ -4,8 +4,7 @@ model{"frame",
     property{"op", "disccord::ws::opcode"},
     property{"s", "disccord::util::optional<uint32_t>"},
     property{"t", "disccord::util::optional<std::string>"},
-
-    field{"d", "web::json::value"},
+    property{"d", "web::json::value"},
```

Comment:
**NOTE:** see https://github.com/FiniteReality/disccord/pull/46/commits/62bf878548ce381e97133f8cd35a66fe05e39819 's commit message for why this changed